sqlglot/dialects/databricks.py

```diff
@@ -51,6 +51,15 @@ class Generator(Spark.Generator):
             exp.ToChar: lambda self, e: self.function_fallback_sql(e),
         }
 
+        PARAMETER_TOKEN = "$"
```

Comment:
why did you add this back in? i think @GeorgeSittas removed it

Reply:
I'll double check.

Reply:
George did remove it, my merge divereged so my mistake. Fixing...